optimize_versioned_sidebars/version-3.9.0-sidebars.json

```diff
@@ -749,50 +795,140 @@
         {
           "type": "link",
           "label": "Introduction",
-          "href": "/docs/8.0/components/connectors/introduction-to-connectors/"
+          "href": "/docs/components/connectors/introduction-to-connectors/"
```

Comment:
These are weird, I don't know why we had 8.0 links in our optimize/3.9.0 navigation.

Reply:
I don't want you to dig too hard, but it may be a copy & paste issue or just someone not fully understanding what that file does 🤷‍♀️